source/agora/node/Ledger.d

```diff
@@ -65,6 +70,30 @@ public class Ledger
         return *this.last_block;
     }
 
+    /***************************************************************************
+
+        Get the array of blocks starting from the provided block height.
+
+        Params:
+            block_height = the starting block height to begin retrieval from
+            max_blocks   = the maximum blocks to return at once
+
+        Returns:
+            the array of blocks starting from block_height,
+            up to `max_blocks`
+
+    ***************************************************************************/
+
+    public Block[] getBlocksFrom ( ulong block_height, size_t max_blocks )
+    {
+        assert(max_blocks > 0);
+
+        if (block_height > this.ledger.length)
```

Comment:
It should be specified in the doc that it's an inclusive range.

Reply:
Inclusive in what regards? It starts at the block height and goes from there. And the ledger always has at least one block (genesis).

Reply:
That the provided `block_height` is included.
So if I want to catch-up, I should send `my_block_height + 1`

Reply:
I mean it literally says that in the docs. `Get the array of blocks starting from the provided block height.` I guess "starting from" is not descriptive enough. I'll add another note.